src/Native/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -41,14 +41,14 @@ enum PAL_GssFlags : uint32_t
 
 struct PAL_GssBuffer
 {
-    uint64_t length;
+    size_t length;
     uint8_t* data;
 };
 
 /*
 Shims the gss_release_buffer method.
 */
-extern "C" void NetSecurityNative_ReleaseGssBuffer(void* buffer, uint64_t length);
+extern "C" void NetSecurityNative_ReleaseGssBuffer(void* buffer, size_t length);
```

Comment:
Managed code at https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.GssBuffer.cs#L17 is still going to pass UInt64. @stephentoub is there any guidance for specifying 64-bit types on ARM?